docs/installation/agent/setup.md

```diff
@@ -1,22 +1,65 @@
 ---
 meta:
-  - name: title
-    content: Fairwinds Insights Agent
   - name: description
-    content: "Fairwinds Insights | Documentation. The Insights Agent runs inside your Cluster, and sends back data to Fairwinds Insights."
-  - name: keywords
-    content: Fairwinds Insights, Kubernetes Audit, Kubernetes configuration validation
+    content: "Fairwinds Insights | Documentation: How to install the in-cluster Agent. "
 ---
-# About
+# Setup
+To get started with the agent, create a new cluster in the Insights UI.
 
-> Installing the agent in many different clusters? Check out the
-> [fleet installation](/run/agent/installation#fleet-installation) docs.
+<div class="mini-img">
+  <img :src="$withBase('/img/new-cluster.png')" alt="new cluster">
+</div>
 
-The Insights Agent runs inside your Cluster, and sends back data to Fairwinds Insights.
+## Install the Agent
 
-The Insights Agent comes with several different open source reporting tools, each of which can be
-configured independently using the Install Hub.
-To read more about the different report types, see the [Reports Section](/run/agent/configuration).
+When you create the cluster, you'll be prompted to install the Insights Agent.
+You'll need to use [Helm](https://helm.sh/) to install - if you'd like to view the Kubernetes
+manifests first, you can use `helm template` to generate the YAML files.
+
+
+## Install Hub
+We recommend using the [Install Hub](/configure/agent/install-hub) to configure the Insights Agent.
+Making changes in the Install Hub will change the provided installation command by setting new
+Helm parameters.
+
+## Fleet Installation
+If you're installing the Insights Agent across a large fleet of clusters,
+it can be tedious to use the UI to create each cluster, then copy out the
+cluster's access token. To better serve customers with a large number of clusters,
+we've created a flow that allows you to easily deploy the Insights Agent across your fleet.
+
+You can use the following commands to install the insights-agent in many clusters,
+changing `insights.cluster` to a unique identifier for each cluster (e.g. generated using
+your kube context).
```

Comment:
Would it be worth it saying that the "cluster_indentifier" here is the name of the cluster that will show up on Insights?